pkg/config/loader.go

```diff
@@ -143,6 +153,13 @@ func (l *Loader) Load() (config *Config, err error) {
 	// Return the created configuration:
 	config = l.config
 
+	// Start/Re-start watching config files for modifications.
+	err = l.ConfigFilesObserver.Watch(l.LoadedFiles)
+	if err != nil {
+		glog.Errorf("Error watching config files: %v.", err)
```

Comment:
In the rest of the code of the project we log messages don't end with punctuation. Can you do the same?